backend/ssm.go

```diff
@@ -0,0 +1,34 @@
+package backend
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"os"
+	"sync"
+)
+
+var ssmClient *ssm.SSM
+var once sync.Once
+
+func GetSsmClient() *ssm.SSM {
+	once.Do(func() {
+		ssmClient = ssm.New(session.Must(session.NewSession()))
```

Comment:
Perhaps move this into `RetrieveFromParameterStore` and use an `if ssmClient != nil` check vs sync.once.